frontend/components/Radio.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+
+    <label class="control-container">Hello
+        <input type="radio" name="asdf" value="123" checked>
```

Comment:
I know that this is very hardcoded for testing purposes, but as it's currently set up, only the last radio control with the given `name` attribute is tab-selectable:

![image](https://user-images.githubusercontent.com/16471311/132431139-b778662b-f665-4eac-bcc5-b6311bd5ae70.png)

Changing the `name` attribute of the second radio input on the page makes them both selectable, but this obviously doesn't work for acutal form creation:

![image](https://user-images.githubusercontent.com/16471311/132431275-28b879f7-d09d-4112-aa79-9f18b3fb70aa.png)

I don't really know why this is happening, but we may want to create a separate issue for it because it's an accessibility problem.

Reply:
Moved to #819